PayoutCurve.md

```diff
@@ -67,54 +69,113 @@ In this section we detail the TLV serialization for a general `payout_function`.
 
 1. type: 42790 (`payout_function_v0`)
 2. data:
-   * [`u16`:`num_pts`]
-   * [`boolean`:`is_endpoint_1`]
-   * [`bigsize`:`event_outcome_1`]
-   * [`bigsize`:`outcome_payout_1`]
-   * [`u16`:`extra_precision_1`]
+   * [`u16`:`num_pieces`]
+   * [`bigsize`:`endpoint_0`]
+   * [`bigsize`:`endpoint_payout_0`]
+   * [`u16`:`extra_precision_0`]
+   * [`payout_curve_piece`:`piece_1`]
+   * [`bigsize`:`endpoint_1`]
```

Comment:
Huge improvement with `payout_curve_piece` enabling it to be a lot more generalizable 👍